weasyl/message.py

```diff
@@ -110,10 +111,19 @@ def select_submissions(userid, limit, backtime=None, nexttime=None):
                 pr.username,
                 ch.settings,
                 we.welcomeid,
-                0 AS subtype
+                0 AS subtype,
+                agg.tags
```

Comment:
Figured this one out; was having issues initially, but I got it to work. I'll be synchronizing it in a bit.